mlir/utils/performance/reportUtils.py

```diff
@@ -21,6 +24,7 @@
                         'X', 'DilationH', 'DilationW', 'StrideH', 'StrideW',
                         'PaddingH', 'PaddingW', 'PerfConfig']
 GEMM_TEST_PARAMETERS = ['DataType', 'Chip', 'TransA', 'TransB', 'G', 'M', 'K', 'N', 'PerfConfig']
+FUSION_TEST_COL = ['TestFile', 'PerfConfig', 'Time']
```

Comment:
This name doesn't seem to be used anywhere else

Reply:
Removed.